tox.ini

```diff
@@ -74,24 +74,16 @@ commands = {envpython} speed.py
 [testenv:codechecks]
 basepython = python3
 deps =
-     pyflakes
+     black==19.10b0
      flake8
 commands =
          flake8 setup.py speed.py src
+         black --check --line-length 79 .
 
 [flake8]
 exclude = src/ecdsa/test*.py
 # We're just getting started. For now, ignore the following problems:
-# E111: indentation is not a multiple of four
-# E114: indentation is not a multiple of four (comment)
-# E226: missing whitespace around arithmetic operator
-# E231: missing whitespace after ','
-# E266: too many leading '#' for block comment
-# E302: expected 2 blank lines, found 1
-# E305: expected 2 blank lines after class or function definition, found 1
+# E203: whitespace before ':' (this needs to be ignored for black compatibility)
 # E501: line too long
```

Comment:
shouldn't we be able to remove this one too?

Reply:
Theres a bunch of lines like
```
_b = 0x051953EB9618E1C9A1F929A21A0B68540EEA2DA725B99B315F3B8B489918EF109E156193951EC7E937B1652C0BD3BB1BF073573DF883D2C34F1EF451FD46B503F00
```
which is something black will not fix. These need to be manually sliced or we have to add `# noqa: E501` comments on these lines (personally would prefer the latter). This could also be a separate MR?

Reply:
those can be translated to `int("0x0519...", 16)` this is the approach we are already using for a lot of constants like this

I guess we can do it as a separate PR...

Reply:
moved  to #187